cmd/otelopscol/main.go

```diff
@@ -57,3 +69,55 @@ func runInteractive(params service.Parameters) error {
 
 	return nil
 }
+
+func setConfigEnvVars() error {
```

Comment:
To be able to cover this function with a unit test, you may split generating of env vars as `map[string]string` and actually setting the env vars in a separate for loop.
Then you can assert that the `map[string]string` contains expected values.